### PR TITLE
Be agnostic from ember-data

### PR DESCRIPTION
**Pull request by GreatWizard**
The aim of this PR is to be able to read some data with a form defitnion and give this directly to formidable-core without any process. With this, no needs to unserialize models, just read an endpoint and transfer data to the form-filler.

Needed to finish first integration in LEAP: https://github.com/peopledoc/employee-app-front/pull/115

### CODE REVIEW

File: addon/helpers/fmd-field.js [hunk 1]
```diff
@@ -58,7 +60,7 @@ export default Helper.extend({
           })
         );
       } else {
-        if (modelType.get('id') === this.get('checkboxField')) {
+        if (modelTypeId === this.get('checkboxField')) {
```
Comment (BlueCutOfficial) [inline addon/helpers/fmd-field.js]: `else if (...)`

File: tests/integration/components/fmd-form-test.js [hunk 2]
```diff
@@ -306,8 +306,8 @@ test('it has fields visible and invisible', function(assert) {
   );
   assert.equal(
     this.$('input[name=lastName]').length,
-    0,
-    'Unknown visible state field is not displayed'
+    1,
+    'Unknown visible state field is displayed'
```
Comment (BlueCutOfficial) [inline tests/integration/components/fmd-form-test.js]: Is it still what we want? I thought you chose to set `isVisible`to `true` by default if it's undefined.

File: addon/helpers/fmd-field.js [hunk 3]
```diff
@@ -40,11 +41,12 @@ export default Helper.extend({
...
   getValidators(model) {
-    let isRequired = model.get('required');
-    let isMultiple = model.get('multiple');
-    let modelType = model.get('type');
-    let validationsKeys = model.get('validationsKeys');
-    let validations = model.getWithDefault('validations', A());
+    let isRequired = get(model, 'required');
+    let isMultiple = get(model, 'multiple');
+    // Firstly, take the normalized by ember-data type, else the payload key
+    let modelTypeId = get(model, 'type.id') || get(model, 'type_id');
```
Comment (BlueCutOfficial) [inline addon/helpers/fmd-field.js]: Is there a way to test both formats in the tests?

**Comment by GreatWizard** [comment]
Updated with tests with data like:
- Field with type: { id : 'dfgdfgfd' }
- Field with type_id: 'dgfdgdf'

**Pull request merged**